scripts/windows/choco/signalfx-agent/signalfx-agent.nuspec

```diff
@@ -9,21 +9,32 @@ The SignalFx Smart Agent is a metric-based agent written in Go that is used to m
 #### Package Parameters
 The following package parameters can be set:
 
- * `/access_token` - The token used to send metric data to SignalFx. Required if C:\ProgramData\SignalFxAgent\token does not exist.
- * `/ingest_url` - Base URL of the SignalFx ingest server. Defaults to 'https://ingest.signalfx.com'.
- * `/api_url` - Base URL of the SignalFx API server. Defaults to 'https://api.signalfx.com'.
+ * `/access_token` - The token used to send metric data to SignalFx. If not specified, the value in the `\ProgramData\SignalFxAgent\token` file (if it exists) will be used. See the **note** below for details regarding the token and starting the SignalFx Smart Agent service after installation.
+ * `/ingest_url` - Base URL of the SignalFx ingest server. If not specified, the value found in the `\ProgramData\SignalFxAgent\ingest_url` file (if it exists) will be used. Otherwise, defaults to `https://ingest.signalfx.com`.
+ * `/api_url` - Base URL of the SignalFx API server. If not specified, the value found in the `\ProgramData\SignalFxAgent\api_url` file (if it exists) will be used. Otherwise, defaults to `https://api.signalfx.com`.
+ * `/install_dir` - Installation directory. Defaults to `\Program Files\SignalFx\SignalFxAgent`.
 
-To pass parameters, use `--params "''"` (e.g. `choco install signalfx-agent [choco options] --params="'/access_token:MY_ACCESS_TOKEN /ingest_url:MY_INGEST_URL /api_url:MY_API_URL'"`).
+To pass parameters, use `--params "''"` (e.g. `choco install signalfx-agent [choco options] --params="'/access_token:MY_ACCESS_TOKEN /ingest_url:MY_INGEST_URL /api_url:MY_API_URL /install_dir:MY_INSTALL_DIR'"`).
 
 To have choco remember parameters on upgrade, be sure to set `choco feature enable -n=useRememberedArgumentsForUpgrades`.
+
+The agent will look for a configuration file at `\ProgramData\SignalFxAgent\agent.yaml`. If one does not already exist during installaion, a sample config file will be copied in place.

```

Comment:
See my previous comments for this section. I think it's identical to one at the beginning of the file.